game-backend/src/game-engine/intent/ConsumeIntent.test.ts

```diff
@@ -0,0 +1,90 @@
+import ConsumeIntent, { FOOD_REGENERATION_VALUE } from './ConsumeIntent';
+import StateBuilder from '../../test-helper/StateBuilder';
+import { State } from '../../../../darwin-types/State';
+import { UserContext } from '../../../../darwin-types/UserContext';
+import { Unit, MAX_HEALTH } from '../../../../darwin-types/game-objects/Unit';
+import { Food } from '../../../../darwin-types/game-objects/Food';
+
+describe('ConsumeIntent', () => {
+  const unitId1 = 'UNIT_ID_1';
+  const unitId2 = 'UNIT_ID_2';
+  const unitId3 = 'UNIT_ID_3';
+  const foodId1 = 'FOOD_ID_1';
+  const foodId2 = 'FOOD_ID_2';
+
+  let state: State;
+  let userContext1: UserContext;
+  let userContext2: UserContext;
+  let userContext3: UserContext;
```

Comment:
What about giving them meaningful names?
e.g.: `normalHealth`, `overflowHealth` and `lowHealth`